app/src/playStore/java/org/fossasia/susi/ai/chat/YouTubeActivity.kt

```diff
@@ -27,22 +27,27 @@ class YouTubeActivity : YouTubeBaseActivity(), YouTubePlayer.OnInitializedListen
     lateinit var playbackEventListener: MyPlaybackEventListener
     private var youtubeId: String = ""
     lateinit var playerView: YouTubePlayerView
+    var apikey = ""
```

Comment:
In Java and Kotlin, you make properties private unless needed otherwise

Make this private lateinit var